libopenage/error/error.h

```diff
@@ -152,7 +152,8 @@ inline std::string no_ensuring_message()
 }
 
 // ENSURE(condition, errormessage << variable << etcetc)
-#define ENSURE(...) do if (unlikely(not OPENAGE_ENS_FIRST(__VA_ARGS__))) throw ::openage::error::Error(MSG(err) OPENAGE_ENS_REST(__VA_ARGS__)); while (0)
+// TODO: Switch to variadic template if needed (or drop the variadic behaviour, if acceptable)
```

Comment:
It may or may not be a macro.

Calling it with just one parameter (the 'condition') should produce a compiler warning.